core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -63,6 +62,9 @@ var ExplorationEditorPage = function() {
     return new ExplorationEditorTranslationTab
       .ExplorationEditorTranslationTab();
   };
+  this.getPreviewTab = function() {
```

Comment:
Where is this used?

Reply:
My bad, it's used right [here](https://github.com/oppia/oppia/pull/8424/files#diff-ed2d9d456667fdbed791c425d044582fR49).